### ISSUE TITLE
Colorinput with eye dropper

### ISSUE THREAD

**Issue by zzzevaka** [scaleflex/filerobot-image-editor#351]
Hello colleagues!

For my use case, I need to pick a colour from the image instead of choosing it from the palette. Default `<input type="color" />` has an eye dropper for such things.

Maybe it is worth to allow replace the custom Colorinput with the default one by using some prop like `useDefaultColorinput`?

What do you think about such a feature? Thank you!

**Comment by yeyu12** [comment]
I also need this feature, is there a plan to implement it?

**Comment by AhmeeedMostafa** [comment]
Yes it is a useful feature, we would ask our @sfx/ui team to consider it in the future updates, keeping the issue opened till we have it thanks for ur suggestion.
